packages/remix-dev/compiler/loaders.ts

```diff
@@ -22,6 +22,7 @@ export const loaders: { [ext: string]: esbuild.Loader } = {
   ".ogg": "file",
   ".otf": "file",
   ".png": "file",
+  ".sql": "text",
```

Comment:
```suggestion
  ".gql": "text",
```
What do you think about also adding `.gql` as text? That will let people import a `.gql` file with a GraphQL query/mutation/subscription and then use it as a variable directly in their code

Reply:
Let's do a separate PR for that.